description: change package name to allow co-existence with original Chromium
author: Wengling Chen <feiyu2817@gmail.com>

---
 chrome/android/chrome_public_apk_tmpl.gni                          |    4 ++--
 chrome/android/java/res_chromium_base/values/channel_constants.xml |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -25,7 +25,7 @@ declare_args() {
   # Android package name to use when compiling the public chrome targets
   # (chrome_public_apk, trichrome_chrome_bundle, etc. as well as the
   # *_bundle variants). This defaults to the Chromium browser package name.
-  chrome_public_manifest_package = "org.chromium.chrome"
+  chrome_public_manifest_package = "org.ungoogled.chromium"
   if (use_stable_package_name_for_trichrome) {
     chrome_public_manifest_package += ".stable"
   } else if (android_channel != "default") {
@@ -33,7 +33,7 @@ declare_args() {
   }
 }
 
-chrome_public_test_manifest_package = "org.chromium.chrome.tests"
+chrome_public_test_manifest_package = "org.ungoogled.chromium.tests"
 
 default_chrome_public_jinja_variables = [
   "channel=$android_channel",
--- a/chrome/android/java/res_chromium_base/values/channel_constants.xml
+++ b/chrome/android/java/res_chromium_base/values/channel_constants.xml
@@ -7,7 +7,7 @@ found in the LICENSE file.
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Chromium</string>
+    <string name="app_name" translatable="false">Ungoogled Chromium</string>
     <string name="bookmark_widget_title" translatable="false">Chromium bookmarks</string>
     <string name="search_widget_title" translatable="false">Chromium search</string>
     <string name="quick_action_search_widget_title" translatable="false">Chromium quick action search</string>
